src/quo2/components/counter/counter.cljs

```diff
@@ -29,11 +33,9 @@
                             (= type :default))
                          colors/white
                          colors/neutral-100))
-        value      (if (integer? value)
-                     value
-                     (js/parseInt value))
-        label      (if (> value 99)
-                     "99+"
+        value      (utils-number/parse-int value)
```

Comment:
Oh that's bizarre behavior. I checked if no exception was thrown, but I didn't check the most important part that you did, which is the actual comparison result. In the JVM I know that an exception is thrown.

> So, what do you think about setting the default-value parameter when using parse-int?

Sounds better @ulisesmac. Thanks for double-checking this.

Reply:
Done! For reference, I used zero as a default, since it's used everywhere in other programming languages. If the code needs a special value to differentiate zero from a bad input, they can pass a keyword as default, which is a common approach in Clojure.

Reply:
Sounds good!